Nos/Views/Discover/DiscoverTab.swift

```diff
@@ -65,19 +65,6 @@ struct DiscoverTab: View {
                     analytics.showedDiscover()
                 }
             }
-            .navigationDestination(for: Event.self) { note in
-                RepliesView(note: note)
-            }
-            .navigationDestination(for: URL.self) { url in URLView(url: url) }
-            .navigationDestination(for: ReplyToNavigationDestination.self) { destination in 
-                RepliesView(note: destination.note, showKeyboard: true)
-            }
-            .navigationDestination(for: Author.self) { author in
-                ProfileView(author: author)
-            }
-            .navigationDestination(for: EditProfileDestination.self) { destination in
```

Comment:
We are losing this one, right? Maybe its fine, I don't see how EditProfile can be directly opened from the DiscoverTab.